intercom/manifest.json

```diff
@@ -0,0 +1,84 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "cacc2ad1-f88b-4306-af8e-90c76e0478a0",
+  "app_id": "intercom",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Gain insights into Intercom Admin activities, Data Events, Conversations, News Items, and Ticket data.",
+    "title": "Intercom",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Intercom Activity",
+        "image_url": "images/intercom_activity.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Intercom Data Event",
+        "image_url": "images/intercom_data_event.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Intercom Conversation",
+        "image_url": "images/intercom_conversation_1.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Intercom Conversation",
+        "image_url": "images/intercom_conversation_2.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Intercom News Item",
+        "image_url": "images/intercom_news_item.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Intercom Ticket",
+        "image_url": "images/intercom_ticket.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Log Collection",
```

Comment:
Please add `Category::Security`

Reply:
Hi @apiazza-dd , Intercom is not a security product it's a customer service platform so we have added collaboration tag.